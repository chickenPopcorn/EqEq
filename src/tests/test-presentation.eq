MyCtx = {
  a = { b + d ^ 2; }
}

d = 5;

MyCtx: find a {
  b = 0;
  print("a = %.0f\n", a);

  b = 5;
  d = 2;
  print("a = %.0f\n", a);

  c = 100;
  a = c;
  print("a = c = %.0f\n", a);
}
