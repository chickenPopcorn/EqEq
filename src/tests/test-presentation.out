a = 25
a = 9
a = c = 100
